templates/project/cf.yml

```diff
@@ -86,15 +88,21 @@ Resources:
     Type: AWS::ECR::Repository
     Properties:
       RepositoryName: {{$project}}/{{$app}}
+      Tags:
+        -
+          Key: ecs-project
+          Value: {{$project}}
+        -
+          Key: ecs-app
```

Comment:
We use `ecs-application` in the LBFargateApp template:
https://github.com/aws/amazon-ecs-cli-v2/blob/c80b45a10e2e1d51cd38ed5252e06fd0abb7bdce/internal/pkg/deploy/cloudformation/stack/tags.go#L9

Could we use that instead?

Reply:
Ah - good point. Updated that.